video_emotion/emotionTagger/face_emotion_extraction.py

```diff
@@ -0,0 +1,61 @@
+import cv2
+import os
+import numpy as np
+from keras.models import load_model
+import glob
+
+def get_labels(dataset_name):
+    if dataset_name == 'fer2013':
+        return {0:'angry', 1:'disgust', 2:'fear',
+                3:'happy', 4:'sad', 5:'surprise',
+                6:'neutral'}
+    else:
+        raise Exception('Invalid dataset name')
+
+def preprocess_input(x, v2=True):
+    x = x.astype('float32')
+    x = x / 255.0
+    if v2:
+        x = x - 0.5
+        x = x * 2.0
+    return x
+    
+def classify_face(face):
+    # parameters for loading data and images
+    dirname = os.path.abspath(os.path.dirname(__file__))
+    emotion_model_path = os.path.join(
+        dirname,
+        "models/emotion_model.hdf5")
+    emotion_labels = get_labels('fer2013')
```

Comment:
Why have this 'fer2013' input hardcoded? It doesn't make sense to have the get_labels function take arguments if it only ever uses one model. And if it is supposed to be possible to extend the get_labels function with more models, then maybe add a comment here, saying why you're using 'fer2013'